### DISCUSSION TITLE
讨论

### DISCUSSION THREAD

**Discussion by zycn0910** [zycn0910/Sign-ZXJY#29] [Announcements]
畅所欲言

**Comment by kfyunyun** [comment]
博主，我是一位实习老师，能不能添加一个批量操作？

**Reply by zycn0910** [reply]
批量添加账号吗？

**Reply by kfyunyun** [reply]
对的，可以吗？我可以赞助> 批量添加账号吗？

**Reply by zycn0910** [reply]
你是想手动输入批量添加，还是从文件（excel表格、txt文档，或者其他类型）导入批量添加？

**Reply by kfyunyun** [reply]
的确是

**Reply by kfyunyun** [reply]
txt

**Comment by 7Fei** [comment]
> 畅所欲言

作者大大，我这今天打不了卡了，求帮忙解决
![屏幕截图 2024-06-15 082115](https://github.com/zycn0910/Sign-ZXJY/assets/156059670/5999407c-4e77-4c33-af2a-9ed1e73e9081)
